ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_fft.py

```diff
@@ -70,3 +70,27 @@ def test_jax_numpy_fftshift(
         x=arr[0],
         axes=None,
     )
+
+
+# ifftshift
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.ifftshift",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), shape=(4,), array_api_dtypes=True
```

Comment:
using `shape=(4,)` seems a bit too restrictive here. The input can really be any n-dim array right?